src/pages/ContentScripts/features/oss-gpt/view.tsx

```diff
@@ -35,6 +35,8 @@ const View = ({ theme, currentRepo, currentDocsName }: Props): JSX.Element => {
   const [inited, setInited] = useState(false);
   const [settings, setSettings] = useState(new Settings());
   const [history, setHistory] = useState<[string, string]>(['', '']);
+  // Toggle to allow resizing of the widget
+  const [resizable, setResizable] = useState(true);
```

Comment:
`resizable` doesn't need to be a React state. `setResizable` is never used.

Actually we can make the chat-widget always resizable so do not need an option to toggle it. Just make it as default, no `if else`.

Reply:
The reason why this needs to be placed in mutation callback is we watch for the changes in rcw-widget-container as this should be fired only when the conversation container is active